zebra-scanner/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+//! Blockchain scanner code for Zebra. 🦓
```

Comment:
Please don't put emoji in source code files, they make some editor / language server combinations panic.

```suggestion
//! Blockchain scanner code for Zebra.
```

Reply:
Really ? It is strange nobody complained ever https://github.com/ZcashFoundation/zebra/blob/main/zebra-state/src/lib.rs#L1

Reply:
Maybe no-one has ever tried to edit that line of that specific file with emacs and rust-analyzer. But it does happen, and lots of people have complained about it in other projects:
https://github.com/ZcashFoundation/zebra/blob/0e00ef4f4f20e40d61f9b7788af8bffb169cbed7/zebra-chain/src/transparent.rs#L58-L62